modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -4,53 +4,86 @@
 
 :_content-type: REFERENCE
 [id="cluster-cloud-controller-config_{context}"]
-= The {rh-openstack} Cloud Controller Manager (CCM) config file
+= The {rh-openstack} Cloud Controller Manager (CCM) config map
 
-A {rh-openstack-first} CCM config file, called `cloud.conf` by convention, comprises several options in an INI format.
+A {rh-openstack-first} CCM config map defines how your cluster interacts with your {rh-openstack} cloud. By default, this configuration is stored under the `cloud.conf` key in the `cloud-conf` config map in the `openshift-cloud-controller-manager` namespace. 
 
-Except when indicated otherwise, these options are mandatory for clusters that run on {rh-openstack}.
+[IMPORTANT]
+====
+The `cloud-conf` config map is generated from the `cloud-provider-config` config map in the `openshift-config` namespace.
 
-[id="ccm-config-global-options"]
-== Global options
+To change the settings that are described by the `cloud-conf` config map, modify the `cloud-provider-config` config map. 
 
-The following options are used for {rh-openstack} CCM authentication with the {rh-openstack} Identity service, also known as Keystone. They are similiar to the global options that you can set by using the `openstack` CLI.
+As part of this synchronization, the CCM Operator overrides some options. For more information, see "The {rh-openstack} Cloud Controller Manager".
+====
 
-|===
-| Option | Description
+For example:
+
+.An example `cloud-conf` config map
+[source,yaml]
+----
+apiVersion: v1
+data:
+  cloud.conf: |
+    [Global] <1>
```

Comment:
~Is `<1>` an asciidoc thing or a typo?~

Later: Ah, it's an asciidoc thing. I see the callout below.